mlos_bench/mlos_bench/config/environments/os/linux/boot/linux-boot-tunables.jsonc

```diff
@@ -0,0 +1,54 @@
+{
+    "linux-kernel-boot": {
```

Comment:
Note: previous "boot" param examples included things like "rootfs" which aren't actually tunable at boot.  This includes things that can be set on the kernel cli (e.g. via grub boot loader).